src/components/o-testimonials/index.scss

```diff
@@ -167,6 +167,14 @@ $o-testimonials-author-position-bottom: -40px;
   animation: fromRight 0.8s;
 }
 
+axa-testimonials[show-all-inline="true"] .o-testimonials__item-author {
```

Comment:
please add a modifier class here instead of going onto the rendered show-all-inline attribute

Reply:
like `.o-testimonials--show-all-inline`

Reply:
would the below be fine?
.o-testimonials--show-all-inline .o-testimonials__item-author {
   // css goes here
}
because I need this css to be applied on optional div in the child fragment.

Reply:
Updated.